lib/api/error-handler.ts

```
@@ -0,0 +1,113 @@
+    message = 'An unexpected error occurred';
+  }
+
+  return createApiErrorResponse(message, status, code);
+}
+
+/**
+ * Wraps an API route handler with error handling
+ */
+export function withErrorHandling<T>(
+  handler: () => Promise<T>,
+  context: string = 'API'
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)